frontend/screens/BrandProfile.js

```diff
@@ -76,8 +76,7 @@ const BrandProfile = ({ navigation }) => {
     <ScrollView style={styles.container}>
       <View style={styles.container}>
         <View style={styles.header}>
-          <TouchableOpacity
-            onPress={() => navigation.navigate("LoginPageBrands")}
+          <TouchableOpacity 
```

Comment:
I am not able to go back after logging in, is this change intentional?

Reply:
you said after login you not want to see login or anything related to signup. so i changed it.